ChangeLog

```diff
@@ -8,7 +8,8 @@ English Dolibarr ChangeLog
 For users:
 ----------
 NEW: Compatibility with PHP 8.2 (with no need to disable warnings)
-NEW: Font param Look and Feel THEME_FONT_FAMILY
+NEW: Font param Look and Feel THEME_FONT_FAMILY (#29302)
+NEW: Param to show main menu logo in color (#29305)
```

Comment:
No need to add entry here. Just put the label to have it as the label of your PR.
ChangeLog will be generated automatically before the release.